build/proj/Build.proj

```diff
@@ -71,6 +71,7 @@
         Test=$(Test);
         Sign=$(Sign);
         IntegrationTest=$(IntegrationTest);
+        VersionsPropsPath=$(RepoRoot)build\import\Versions.props;
```

Comment:
This is required by the toolset? I don't see any other use of `VersionsPropsPath` in the PR.

Reply:
Yes, it's needed as the default has changed and used by the toolset.